docs/application/dotnet/tutorials/certificates/creating-certificates.md

```diff
@@ -0,0 +1,116 @@
+
+![Author Certificate Advanced options](media/certification_guide8-2.png)
+
+The same signing key helps in maintaining secure peer authentication between the Tizen Gear App and the Android mobile app.
+You can find out more details from the [Accessory SDK](https://developer.samsung.com/galaxy/accessory). In case of selecting a Samsung author certificate, you can just renew the certificate with the same information. This renewed certificate will be the same as your original RSA key so upgrading your application is guaranteed.
+
+### 2. Information of the author certicate
+
+Enter the author name and password. It is recommended to set a strong password and not to forget it . If you want to use the same password for the distributor certificate in the next step, check the box below. `Additional Fields` contains more CSR(Certificate Signing Request) information to input, but are not mandatory.
+
+![Create a new author certificate](media/certification_guide9.png)
+
+### 3. Sign in to Samsung Account
+
+At the Next step, a Samsung Account sign-in pops up. Sign in or Create a account. ( To get a certificate, you should have a Samsung Account. If you don't have it yet, please create a new account.)
+
+![samsung account](media/certification_guide11.png)
+
+### 4. Backup created Author certificate
+
+After a few seconds from signing-in you can get the author certificate.
+
+![Author certificate is complete](media/certification_guide12.png)
+
+The author certificate should be secure. So we recommend making a backup. You can specify the backup location by clicking `Browse` button.
+
+Be careful because if you lose the author certificate you will not be able to upgrade your application signed with the same certificate.
+
+The `Next` step is creating distributor certificate.
+
+## Create a new distributor certificate
+
+### 1. Create new or select existing distributor certiicate
+
+Select `Create a new distributor certificate`. In case of `Select an existing distributor certificate`, see the [Managing Certificate Profile](managing-certificate-profile.md).
+
+![Distributor certificate](media/certification_guide13.png)
+
+### 2. Information of the distributor certicate
+
+Select the `Privilege` level for the distributor certificate. To ensure that implemented APIs work in the device, the application should be signed with distributor certificate which has appropriate privilege level.
+
+![Set the Privilege and DUID](media/certification_guide15.png)
+
+> **Privileges**  
+> To effectively protect the device system and user private data, the Tizen security architecture is based on privileges and application signing of the Linux basic security model, which includes process isolation and mandatory access control. The users must be able to grant privileges for security-sensitive operations  
```

Comment:
Can we rephrase as:
To protect the device system and user private data, the Tizen security architecture is based on privileges and application signing of the Linux basic security model. Linux basic security model includes process isolation and mandatory access control. The users must have options to grant privileges for security-sensitive operations.

I am not clear with "application signing". Hence, left without rephrasing. Please clarify.

As the sentence was too wordy, I tried to split the sentence.
Please verify whether the technical meaning is altered.

Reply:
'Application signing' is a proper word.

https://developer.tizen.org/development/training/native-application/understanding-tizen-programming/application-signing-and-certificates